<launch>
    <param name="GPSPR2Plugin/type" value="gps_agent_pkg/GPSPR2Plugin" />
    <!-- rosparam command="load" file="$(find gps_agent_pkg)/launch/pr2_head_controller.yaml" -->
    <rosparam command="load" file="$(find gps_agent_pkg)/launch/pr2_left_controller.yaml" />

    <!-- controller params -->
    <param name="GPSPR2Plugin/root_name" value="torso_lift_link" />
    <param name="GPSPR2Plugin/active_tip_name" value="l_gripper_tool_frame" />
    <param name="GPSPR2Plugin/passive_tip_name" value="r_gripper_tool_frame" />

    <!-- kalman filter matrices -->
    <param name="GPSPR2Plugin/encoder_filter_params" textfile="$(find gps_agent_pkg)/encoder_filter_params.txt" />

    <!-- Spawn Controller -->
    <node name="controller_spawner"
         pkg="pr2_controller_manager" type="pr2_controller_manager" output="screen"
         args="spawn GPSPR2Plugin" />

</launch>
